tests/integration_tests/test_applications.py

```diff
@@ -401,16 +401,34 @@ def prepare_one_file(file_name):
     prepare_one_file("MLTdata-preproduction.ecsv")
 
     def make_command(app, args):
-        cmd = "python simtools/applications/" + app + ".py"
+        if app.find("simtools-") < 0:
+            cmd = "python simtools/applications/" + app + ".py"
+        else:
+            cmd = app
         for aa in args:
             aa = aa.replace(
                 "TESTMODELDIR", str(io_handler.get_output_directory(dir_type="model", test=True))
             )
             cmd += " " + aa
         return cmd
 
+    def get_application_name(app):
+        """
+        Get the application name from the app string.
+        Return name of command line tool in case pip installation
+        is used.
+        """
+
+        # check if conda is installed on this machine
+
+        if shutil.which("conda") is not None:
```

Comment:
Is this the best way to check if we are in the "dev" case? Probably it is, just wondering if you considered other options.

Reply:
As discussed in your office: this is a simple solution for running integration tests on the github runners (with a clean environment). I assume that if we make the move away for the dev environment from conda, it will be straight forward to change this one line?